src/test/java/org/mifos/integrationtest/cucumber/stepdef/VoucherManagementStepDef.java

```diff
@@ -313,11 +322,7 @@ public void createVoucher() {
         mockServerStepDef.startStub("/createVoucher", PUT, 200);
         mockServerStepDef.startStub("/activateVoucher", PUT, 200);
         iCallTheVoucherCreateAPIWithExpectedStatusOf(202, "/createVoucher");
-        try {
-            Thread.sleep(5000);
-        } catch (InterruptedException e) {
-            throw new RuntimeException(e);
-        }
+        await().atMost(5000, MILLISECONDS).pollInterval(1, MILLISECONDS).until(() -> true);
```

Comment:
the wait condition is not clear, hard coding it as true will not wait for the actual condition